builder.js

```diff
@@ -885,6 +885,37 @@ function shareBuild() {
     }
 }
 
+function saveBuild() {
+    if (player_build) {
+        let savedBuilds = window.localStorage.getItem("builds") === null ? {} : JSON.parse(window.localStorage.getItem("builds"));
+        let saveName = document.getElementById("build-name").value;
+        if ((!Object.keys(savedBuilds).includes(saveName) || document.getElementById("saved-error").textContent !== "") && location.hash !== "") {
+            savedBuilds[saveName] = location.hash.replace("#", "");
```

Comment:
This feels dangerous -- what if location.hash isn't a standard build? since the site can (sometimes) not crash when given a url that isn't a build (it'll spit out garbage though)

maybe its alright but I'd feel safer if this was using whatever string was generated by the build encoder directly (this is a reminder to myself also to make sure we don't directly use location.hash elsewhere).

Reply:
Fixed in new commit.